collections/_article/submissions-open-godot-2024-showreel.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Submissions open for Godot 2024 showreel!"
+excerpt: "Showcase your Godot creations in 2024! We're inviting submissions of short videos featuring your published or in-progress Godot games or tools. Don't miss this chance to be a part of it!"
+categories: ["news"]
+author: Emi
+image: /storage/blog/covers/godot-showreel-2024.webp
+image_caption_description: Illustration by André Marí Coppola
+date: 2024-09-05 15:00:00
+---
+
+Showcase your Godot Engine 2024 creations! We invite you to submit videos featuring your published or in-progress games or tools so we can make the Godot 2024 Showreel.
+
+The showreels are an informative and celebratory showcase, to highlight the wide array of projects developed using the Godot Engine. If you want to participate and share your Godot-made game, app or tool submit your entry starting today.
+
+## How to submit
+You can submit your entry in our [Godot Showreel website](https://showreel.godotengine.org/). You can use your Godot account or any other account (GitHub, Discord, etc.) to log in. Before submitting, please read the [submission guidelines](https://showreel.godotengine.org/about).
+
+- ✅ **[Submit your entry](https://showreel.godotengine.org/)**
+- 📋 **[Submission guidelines](https://showreel.godotengine.org/about)**
+- 📅 **Submissions close on November 1st 2024 at 10:00 UTC**
+
+## Changes from previous years
+
+This year, the Godot Foundation will be selecting some games to be featured as well as the ones submitted by the community. This will be done due to the confusion caused by big titles not being present in past showreels because the developers forgot to submit or were not aware of our efforts. It shouldn't change anything for the submission process, but we wanted to make sure we are transparent about the changes.
+
+## Need inspiration?
+Check out our [showreel playlist](https://www.youtube.com/watch?v=UAS_pUTFA7o&list=PLeG_dAglpVo6EpaO9A1nkwJZOwrfiLdQ8) for some creative sparks and see what makes a great submission!
```

Comment:
This starts with the second video in the playlist (2022), I think we should adjust the link to start with the first video (2023):
```suggestion
Check out our [showreel playlist](https://www.youtube.com/watch?v=W1_zKxYEP6Q&list=PLeG_dAglpVo6EpaO9A1nkwJZOwrfiLdQ8) for some creative sparks and see what makes a great submission!
```